Mathlib/Order/Filter/Bases.lean

```diff
@@ -466,7 +466,7 @@ theorem HasBasis.le_iff (hl : l.HasBasis p s) : l ≤ l' ↔ ∀ t ∈ l', ∃ i
 theorem HasBasis.le_basis_iff (hl : l.HasBasis p s) (hl' : l'.HasBasis p' s') :
     l ≤ l' ↔ ∀ i', p' i' → ∃ i, p i ∧ s i ⊆ s' i' := by
   simp only [hl'.ge_iff, hl.mem_iff]
-#align filter.has_basis.le_basis_iff Filter.HasBasis.le_basis_iffₓ
+#align filter.has_basis.le_basis_iff Filter.HasBasis.le_basis_iff
```

Comment:
Why this change? Presumably the `ₓ` was added for a good reason.

Reply:
Perhaps there is a good reason indeed. I don't know the meaning of that x.
The issue that prompted the change is that in the main file of this PR there was an error because the lean 3 lemma `le_basis_iff` was translated to `le_basis_iffₓ`, which is apparently not a lemma that exists? The lemma above the align is `le_basis_iff` without the x and using that lemma works, so I thought the align was a mistake?

Reply:
Aha. That sounds like a bug in mathport. The `ₓ` in an align statement signifies that the translation isn't exactly faithful. Most cases I've seen so far are because of reordered universes.
But it should not appear in places where the lemma is used...
```suggestion
#align filter.has_basis.le_basis_iff Filter.HasBasis.le_basis_iffₓ
```